src/components/main/actionsPanel/workspaceTreeView/hooks/useNodeActionsHandler.ts

```diff
@@ -329,6 +331,13 @@ export const useNodeActionsHandler = ({
       },
     );
     removeInvalidNodes(...uids);
+    const { _fileTree } = await loadLocalProject(
```

Comment:
@VictoriaShyika 
this part is for syncing file-tree after removing files/directories, correct?
can you build some function like `reloadCurrentProject` which we can use everywhere.
am asking this since we will need this for most of file-actions. and not sure yet, but also we can have file-tree reload timer in the future. so it would be great if we have such function in the main context.
does it make sense for you? just let me know your thoughts. thanks.

Reply:
@jasonliu7040  Yes, its part is for syncing file-tree after removing.
I think so, `reloadCurrentProject` function make sense